test/basic001/reg005.dSYM/Contents/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file from the PR and, if possible, add a pattern for it to `.gitignore`. No need for us to have this kind of thing in the repo!